pinot-controller/src/main/java/org/apache/pinot/controller/BaseControllerStarter.java

```diff
@@ -903,7 +912,7 @@ protected List<PeriodicTask> setupControllerPeriodicTasks() {
         _controllerMetrics, _executorService, _connectionManager);
     periodicTasks.add(responseStoreCleaner);
     PeriodicTask resourceUtilizationChecker = new ResourceUtilizationChecker(_config, _connectionManager,
-        _controllerMetrics, _diskUtilizationChecker, _executorService, _helixResourceManager);
+        _controllerMetrics, _utilizationCheckers, _executorService, _helixResourceManager);
```

Comment:
Should we consider passing in `_resourceUtilizationManager`?

Reply:
why would we do that? the manager is just for checking the status, whereas the checker updates the in-memory state

Reply:
From high level, manager should handle things related to checkers. E.g. here we don't need to update the state when checker is not enabled

Reply:
true, how about i make this as a follow-up change since it is really more about the existing framework?